### ISSUE TITLE
Image handling

### ISSUE THREAD

**Issue by mpmdean** [NSLS-II-SIX/sixtools#14]
The current version of sixtools leaves the db registry in its default mode of returning a pims ImageStack and then applies a pims pipeline to the datastream returned.

Is this the best way of doing things?

Should we, for example, use dask?

**Comment by mpmdean** [comment]
In the near future databroker will return xarray s

**Comment by danielballan** [comment]
Yep! And, as you might also already know, dask was originally developed with the xarray user community, so xarray and dask cooperate very well. Databroker will give you a choice of xarray-of-numpy-arrays and xarray-of-dask-arrays.
